spec/controllers/activities_controller_spec.rb

```diff
@@ -101,6 +101,14 @@
         expect(response).to be_successful
         expect(response).to render_template "index"
       end
+
+      describe "with not existent user_id" do
+        it "renders 404" do
+          get "index", params: { project_id: project.id, user_id: 123123123 }
+          expect(response).to have_http_status(:not_found)
+          expect(response).to render_template "common/error"
+        end
```

Comment:
Can you add a test with an inactive user instead? It would better reflect what happened in production.

Reply:
Yes, of course.